docs/source/common/trainer.rst

```diff
@@ -695,6 +696,9 @@ See Also:
 gpus
 ^^^^
 
+.. warning:: Setting `Trainer(gpus=x)` is deprecated in v1.6 and will be removed"
+    in v2.0. Please use `Trainer(accelerator='gpu', devices=x)` instead.
+
```

Comment:
this is not yet the case. let's not add it until we have the deprecation warnings